format_checker/forked-projects.json

```diff
@@ -463,4 +464,4 @@
   "https://github.com/zeroturnaround/gradle-jrebel-plugin": "unforked",
   "https://github.com/zhangxd1989/spring-boot-cloud": "unforked",
   "https://github.com/zhurlik/gradle-jboss-modules-plugin": "unforked"
-}
```

Comment:
Please don't make these unnecessary changes.

Reply:
I wonder how do I cancel the change?

Reply:
@zzjas can help in class today if needed.